packages/split-button/src/SplitButton/SplitButton.spec.tsx

```diff
@@ -251,9 +265,9 @@ describe('packages/split-button', () => {
         await waitForElementToBeRemoved(menuEl);
         expect(menuEl).not.toBeInTheDocument();
       });
-      test('Returns focus to trigger {usePortal: true}', async () => {
+      test('Returns focus to trigger {renderMode: "portal"}', async () => {
         const { openMenu, menuTrigger } = renderSplitButton({
-          usePortal: true,
+          renderMode: "portal"
```

Comment:
Fix to account for the breaking change introduced in https://github.com/mongodb/leafygreen-ui/pull/2520